css/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <div>This is a Landing Page</div> 
+      <div><img id="header-img" src="Images/IMG_5372.JPG" alt="Photo of a puppy"/ width="300px"></div>
+      
+      <div><nav id="nav-bar">
```

Comment:
"nav" tag is enough, you don´t need the "div" tag here...remember it!

Reply:
- [ ] Hey Juan ✋, Please make sure to make your comments sound more polite and less of an order, and always make sure you provide a reason as to why you request changes. I like what you've done with the rest of your comments they're more polite and detailed in the changes you requested.
- [ ] The comment you made above doesn't sound that **polite** so kindly rephrase it to sound more gracious (avoid the use of exclamation marks as they indicate strong emotions) and also make sure you provide a **"Why"** for your requested change. ✔